buoi_8_js/btvn_4.js

```diff
@@ -0,0 +1,17 @@
+/*Bài 4: Sử dụng map để chuyển đổi key của object thành camelCase*/
+
+// đọc tệp JSON là sử dụng require() => đọc và phân tích cú pháp tệp JSON và trả về một đối tượng JSON
+const users = require('./data.json')
+
+let usersTwo = users.map((user, index, users) => {
```

Comment:
- Chỗ này dùng const được không
- Thứ hai là làm ntn cũng được nhưng mà nên nghĩ bài toán tổng quát hơn, là em không biết dữ liệu ntn chỉ biết là phải convert sang camelcase cho key => Từ đó em sẽ học được tư duy cho bài toán tổng quát => Từ đó em sẽ học được cách xây common cho dự án nhé